src/commands/app/deploy.js

```diff
@@ -324,6 +342,10 @@ Deploy.flags = {
     default: true,
     exclusive: ['action']
   }),
+  'force-deploy': flags.boolean({
+    description: '[default: false] Force deploy extensions regardless of published status or workspace',
+    default: false
+  }),
   'force-publish': flags.boolean({
     description: 'Force publish extension(s) to Exchange, delete previously published extension points',
```

Comment:
Not part of your PR but would be good to add the missing "[default:false]" in the description for consistency here too.

Reply:
Agreed, seems like a good standard to have. Change incoming!